api/src/views/confirmation.hbs

```diff
@@ -59,15 +59,19 @@
 
             <h2>{{t "confirmation.needToMakeUpdates"}}</h2>
             <p>{{t "leasingAgent.contactAgentToUpdateInfo"}}</p>
+            {{#if (or listing.leasingAgentName listing.leasingAgentTitle listing.leasingAgentPhone listing.leasingAgentEmail)}}
```

Comment:
suggestion: Can you add a test to `email.service.spec` that doesn't have leasing agent details to verify that this works?